src/NuGetForUnity/Editor/NugetConfigFile.cs

```diff
@@ -74,6 +76,22 @@ public class NugetConfigFile
         /// </summary>
         public bool ReadOnlyPackageFiles { get; set; }
 
+        /// <summary>
+        ///     Gets or sets absolute path to directory containing packages.config file.
+        /// </summary>
+        public string PackagesConfigDirectoryPath { get; set; }
+
+        public string RelativePackagesConfigPath
```

Comment:
```suggestion
        /// <summary>
        ///     Gets or sets the relative path to directory containing packages.config file. The path is relative to the folder containing the 'NuGet.config' file.
        /// </summary>
        public string RelativePackagesConfigDirectoryPath
```

Reply:
Added this but without `sets` as Rider reminded mi the setter is private and therefore shouldn't be a part of the summary 😄